README.md

```diff
@@ -227,6 +227,8 @@ To [deploy financial-demo](https://rasa.com/docs/rasa/user-guide/how-to-deploy/)
 [one line deploy script](https://rasa.com/docs/rasa-x/installation-and-setup/one-line-deploy-script/) for Rasa X. As part of the deployment, you'll need to set up [git integration](https://rasa.com/docs/rasa-x/installation-and-setup/integrated-version-control/#connect-your-rasa-x-server-to-a-git-repository) to pull in your data and
 configurations, and build or pull an action server image.
 
+For running Rasa X in local mode, you can refer to our guided workflow in the [Wiki page](https://github.com/RasaHQ/financial-demo/wiki/Using-Rasa-X-with-the-Financial-Demo) for how to get started.
+
```

Comment:
How about making a note of this closer to the top? Something like "You can try out the bot locally, or try it out in Rasa X with pre-loaded conversations!" Under "run the bot"

Reply:
A link to the install instructions for local mode would also be helpful - or you could even include the one-liner in the readme